packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -490,6 +491,14 @@ export default class Meeting extends StatelessWebexPlugin {
      * @memberof Meeting
      */
     this.callEvents = [];
+    /**
+     * There is a pending floor requested by the user
+     * @instance
+     * @type {floorGrantPending}
```

Comment:
```suggestion
     * @type {Boolean}
```

Reply:
`floorGrantPending` isn't a type, from the default value it is appears to be a boolean